src/event/trigger.js

```diff
@@ -93,7 +96,7 @@ jQuery.extend( jQuery.event, {
 		// Fire handlers on the event path
 		i = 0;
 		while ( ( cur = eventPath[ i++ ] ) && !event.isPropagationStopped() ) {
-
+			lastElement = cur;
```

Comment:
You can reduce the size increase of jquery.min.gz by using `eventPath[ i - 2 ]` below rather than introducing a new variable.